src/js/components/Box/doc.js

```diff
@@ -168,10 +168,11 @@ export const doc = Box => {
         'horizontal',
         'vertical',
         'all',
+        'between',
       ]),
       BORDER_SHAPE,
       PropTypes.arrayOf(BORDER_SHAPE),
-    ]).description('Include a border.'),
+    ]).description('Include a border. You must have "gap" to use "between"'),
```

Comment:
I wonder if we can add some explanation about `between`, something like 'between will place a border between child elements' or '...where a gap is applied'.

Reply:
agreed, done